core/services/user_service/views/user.py

```diff
@@ -42,12 +47,25 @@ async def login(data: LoginRequest):
 @router.post("/refresh", response_model=RefreshAccessResponse)
 async def refresh_access(data: RefreshAccessRequest):
     try:
-        access_token, refresh_token = auth_controller.refresh_access_token(
-            data.refresh_token
-        )
+        access_token, refresh_token = \
```

Comment:
Maybe call it `old_access_token` or smth?

Reply:
This occurence is actually the new_access_token. I'm changing it to new_access_token and old_access_token for better readability.